dialogflow/stopwatch/Stopwatch/DialogflowViewController.swift

```diff
@@ -39,6 +39,10 @@ class DialogflowViewController: UIViewController {
   var  textFieldBottomConstraint: NSLayoutConstraint!
   let intentTextFieldController: MDCTextInputControllerOutlined
   var tableViewDataSource = [[String: String]]()
+  lazy var alert : UIAlertController = {
+    let alert = UIAlertController(title: "Alert", message: "Please wait a moment", preferredStyle: .alert)
```

Comment:
Is this alert when we're waiting for a new token?
If so, let's clarify the message with: "Retrieving token ..."